foxpuppet/expected.py

```diff
@@ -10,7 +10,7 @@ class new_browser_window_is_opened(object):
     """An expectation for checking that a new window is found.
 
     Returns:
-        A BrowserWindow object.
+        :py:class:`BrowserWindow`: Browser window.
```

Comment:
Yeah how it would be rendered to the user. But I guess they would understand it or if not the docs online would be good. Or maybe we could have a doc build step so they could have local access to them if they have a browser.